profiles/Bylickilabs.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/109308073/209451382-c2c90f77-0c2a-45dc-bd06-1f5418910dd9.jpg",
+  "issueId": 1589,
```

I believe the issue ID here must be #1588.
```suggestion
  "issueId": 1588,
```